modules/cnf-topology-aware-lifecycle-manager-cluster-logging-6x-update-policygenerator.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="talm-cluster-logging-6x-update-policygenerator_{context}"]
+= Removing Cluster Logging Operator artifacts with {policy-gen-cr} CRs
+
+When you update to Cluster Logging Operator version 6.x from version 5.x, you must add a cleanup policy that removes the old Operator artifacts from deployed clusters after creating the new API custom resources (CRs).
+
+.Procedure
+
+. Add the `acm-group-du-clo5-cleanup.yaml` file in the git repository that contains your other `PolicyGenerator` CRs.
+
++
+.Example cleanup files
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: group-du-sno-clo5-cleanup
+placementBindingDefaults:
+  name: group-du-sno-clo5-cleanup-placement-binding
+policyDefaults:
+  namespace: ztp-group
+  # Allow this policy to be unbound from clusters based on a custom label
+  placement:
+    labelSelector:
+      matchExpressions:
+      - key: group-du-sno
+        operator: Exists
+      - key: du-profile
+        operator: In
+        values: ["latest"]
+      - key: clo5-cleanup-done
+        operator: DoesNotExist
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 10m
+    noncompliant: 10s
+policies:
+- name: group-du-sno-clo5-cleanup
+  policyAnnotations:
+    ran.openshift.io/ztp-deploy-wave: "11" <1>
+  manifests:
+    - path: source-crs/ClusterLogging5Cleanup.yaml
+----
+<1> Default value is `11`. Ensure that the wave of `group-du-sno-clo5-cleanup` policy is after the `group-du-sno` configuration policy, which is wave `10` by default.
+
+. Update the binding rules in the cleanup `PolicyGenerator` CR as needed to match the labels in use for the associated fleet of clusters.
```

Comment:
> cleanup `PolicyGenerator` CR

Again, will folks know what this is? Is it the `acm-group-du-clo5-cleanup.yaml` file?

Reply:
I think it's obvious from the context but I clarified, just to be sure